# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/01/03 11:00:47+01:00 perex@suse.cz 
#   [ALSA] opl4 depends on opl3
#   
#   OPL3
#   Compiling OPL4 doesn't include the snd-opl3-lib. This change in the
#   Makefile enables now the compiling of snd-opl3-lib.o for OPL4 soundcards.
#   
#   Signed-off-by: Martin Langer <martin-langer@gmx.de>
#   Signed-off-by: Takashi Iwai <tiwai@suse.de>
# 
# sound/drivers/opl3/Makefile
#   2004/12/06 12:46:27+01:00 perex@suse.cz +1 -0
#   [ALSA] opl4 depends on opl3
#   
#   D:2004/12/06 19:46:27
#   C:OPL3
#   F:drivers/opl3/Makefile:1.22->1.23 
#   L:Compiling OPL4 doesn't include the snd-opl3-lib. This change in the
#   L:Makefile enables now the compiling of snd-opl3-lib.o for OPL4 soundcards.
#   Signed-off-by: Martin Langer <martin-langer@gmx.de>
#   Signed-off-by: Takashi Iwai <tiwai@suse.de>
# 
diff -Nru a/sound/drivers/opl3/Makefile b/sound/drivers/opl3/Makefile
--- a/sound/drivers/opl3/Makefile	2005-01-22 22:49:12 -08:00
+++ b/sound/drivers/opl3/Makefile	2005-01-22 22:49:12 -08:00
@@ -18,4 +18,5 @@
 sequencer = $(if $(subst y,,$(CONFIG_SND_SEQUENCER)),$(if $(1),m),$(if $(CONFIG_SND_SEQUENCER),$(1)))
 
 obj-$(CONFIG_SND_OPL3_LIB) += snd-opl3-lib.o
+obj-$(CONFIG_SND_OPL4_LIB) += snd-opl3-lib.o
 obj-$(call sequencer,$(CONFIG_SND_OPL3_LIB)) += snd-opl3-synth.o
